proto/umee/leverage/v1/tx.proto

```diff
@@ -257,12 +257,14 @@ message MsgGovUpdateRegistry {
   option (gogoproto.goproto_stringer) = false;
   option (cosmos.msg.v1.signer)       = "authority";
 
-  // authority is the address of the governance account.
+  // field 2 (title) is removed. Proposal metadata should be used instead.
+  reserved 2;
+
+  // authority is the address of the governance account or the Emergency Group.
   string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  // Deprecated: use proposal metadata instead.
-  string title = 2 [deprecated = true];
-  // Deprecated: use proposal metadata instead.
-  string description = 3 [deprecated = true];
+  // description motivating the change. Should be used only when executing by the
+  // Emergency Group. Otherwise the x/gov Proposal metadata should be used.
+  string description = 3;
```

Comment:
What other options do we have for recording emergency group intentions? transaction memo field?

Would description be any more visible / permanent in storage?

Reply:
Would be ideal if we could eliminate this field, as Sai said